sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -298,6 +323,11 @@ private[ui] class ExecutionPagedTable(
 
     val headerRow: Seq[Node] = {
       executionHeadersAndCssClasses.map { case (header, sortable) =>
+
+        val headerTooltip = {
+          headerNameWithTooltips(header)
+        }
+
```

Comment:
Let's replace the above 5 lines with the following one liner.
```scala
val headerTooltip = headerNameWithTooltips(header)
```

Reply:
ok